vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -7,16 +7,18 @@ def textual_group_properties
     items = %w(
       namespace
       name
-      port
       creation_timestamp
       resource_version
       session_affinity
-      portal_ip
-      protocol
-      container_port)
+      portal_ip)
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_port_configs
```

Comment:
Is this a new group in the summary page?  If so, this should be named `textual_group_port_configs` and needs exposure from the view, which I don't see in this PR, so will be needed.

Can you attach a screenshot showing these changes?